sklearn/manifold/_t_sne.py

```diff
@@ -715,15 +739,23 @@ def _fit(self, X, skip_num_points=0):
                     print("[t-SNE] Computing pairwise distances...")
 
                 if self.metric == "euclidean":
+                    # Euclidean is squared here, rather than using **= 2,
+                    # because euclidean_distances already calculates
+                    # squared distances, and returns np.sqrt(dist) for
+                    # squared=False.
                     distances = pairwise_distances(X, metric=self.metric,
                                                    squared=True)
                 else:
+                    # Euclidean has separate call as it's slower for n_jobs>1
```

Comment:
that's not Euclidean, is it?

Reply:
Ah, yes, my placement of this comment made the message unclear. It was meant to suggest something along the lines of 
```
# The Euclidean case is separate from this call (see above) as it's slower for n_jobs>1
```
Thank you for pointing this out! I'll change it to be more clear. :)